topics/racket-languages/index.md

```diff
@@ -0,0 +1,14 @@
+---
+aliases: racketlanguages
+created_by: PLT Inc.
+display_name: Racket Languages
+github_url: https://github.com/racket
+logo: racket.png
+related: drracket, racket
+released: 1995
+short_description: The 'racket-languages' topic is for programming languages implemented on the Racket platform.
+topic: racket-languages
+url: http://beautifulracket.com
+wikipedia_url: https://en.wikipedia.org/wiki/Racket_features#Language_Extensions
+---
+The topic 'racket-languages' indicates the repository includes languages made with the Racket language toolchain.
```

Comment:
How would you feel about using the description from the [Racket website](http://racket-lang.org/)?

```suggestion
Racket is a general-purpose programming language as well as the world’s first ecosystem for language-oriented programming.
```